_case_studies/11_targeting_variants_for_maximum_impact.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Targeting variants for maximum impact using the CausalTune library
+layout: page
+description: >-
+  This tutorial provides an introduction to causal AI using the CausalTune library in Python. It shows a practical example and the use of the ERUPT metric.
```

Comment:
I'd recommend emphasizing what the reader will learn and be able to do after reading this tutorial.  If the ERUPT metric is a key part of that, maybe briefly mention it like "This tutorial provides an introduction to [doing ABC] using the ERUPT metric and the CausalTune library in Python.

Alternatively, leave the ERUPT metric out of the short description and reference it in the summary below where there's a little more room to define it.

Reply:
Changed to "This tutorial provides an introduction to improving business metrics using the ERUPT metric and the CausalTune library in Python. It shows a practical example and the use of the ERUPT metric for optimizing clickthrough rates."